server/src/main/java/org/elasticsearch/index/mapper/TimeSeriesParams.java

```diff
@@ -25,18 +25,34 @@ private TimeSeriesParams() {}
 
     public enum MetricType {
         GAUGE(new String[] { "max", "min", "value_count", "sum" }),
-        COUNTER(new String[] { "last_value" });
+        COUNTER(new String[] { "last_value" }),
+        POSITION(new String[] { "last_value", "geo_line" }, false);
```

Comment:
We are not supporting downsampling at this stage so why not have here at empty array for now?

Reply:
OK. I did discuss this with Martijn, and he felt like these strings might not have been kept up to date anyway, so I was using this as a reminder of what to support. But I'm happy to remove it for cleanliness sake.

Reply:
I removed them for now, and we can add the right stuff when we do the downsampling work.